integreat_cms/core/logging_filter.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from django.conf import settings
+from path.to.middleware import get_current_user
+import logging
+import threading
+
+class DebugRequestFilter(logging.Filter):
+    def filter(self, record):
+        request = threading.local()
+        if request and request.user.is_authenticated and request.user.is_superuser:
+            if request.GET.get('debug') == '1' and record.levelno >= logging.INFO:
+                record.levelno = logging.WARNING
+                record.levelname = 'WARNING'
```

Comment:
By default, `INFO` has a value of 20 and `WARNING` has a value of 30. Those can be changed I think and new levels might be introduced. You likely rather want to ensure the log level is at most `WARNING`
```suggestion
            if request.GET.get('debug') == '1' and record.levelno > logging.WARNING:
                record.levelno = logging.WARNING
                record.levelname = 'WARNING'
```